### PR TITLE
refactor send email functionality

### PR DESCRIPTION
**Pull request by SimonLab**
## Do not merge yet, it is not ready

On this PR we are authenticate the user to use the gmail api only once instead on each email sent. see [319#issuecomment-198350683](https://github.com/FAC-GM/app/issues/319#issuecomment-198350683)

### CODE REVIEW

File: lib/email.js [hunk 1]
```diff
@@ -130,18 +145,21 @@ exports.register = function (server, options, next) {
         var currentDate = new Date();
         sentEmail.sentAt = currentDate.getDate() + '-' + ("0" + (currentDate.getMonth() + 1)).slice(-2) + '-' + currentDate.getFullYear();
 
+        //authenticate to use gmail api
+
+        oauth2Client.setCredentials(profile.tokens);
```
Comment (SimonLab) [inline lib/email.js]: We setting the credentials on the handler instead of the function send_email

Comment (nelsonic) [inline lib/email.js]: @SimonLab could we PR this change into https://github.com/nelsonic/sendemail-gmail/ ?

**Comment by nelsonic** [comment]
@SimonLab I just checkout out the `email-auth` branch and ran `npm run dev`
when I try to email Maria Dolores I get a 500. see: https://github.com/FAC-GM/app/issues/375
I've tried debugging it locally but don't see why the ES `client.get` is returning `found:false` ...
do you have time to *investigate* this?

**Comment by SimonLab** [comment]
I'm closing this PR as it is quiet old now and it doesn't match the master branch anymore. A new PR will be created based on the branch https://github.com/FAC-GM/app/tree/clean-email-handler

**Pull request closed without merging**